src/EditorFeatures/Core/Implementation/CodeFixes/ICodeFixService.cs

```diff
@@ -11,6 +11,7 @@ namespace Microsoft.CodeAnalysis.CodeFixes
     internal interface ICodeFixService
     {
         Task<ImmutableArray<CodeFixCollection>> GetFixesAsync(Document document, TextSpan textSpan, bool includeSuppressionFixes, CancellationToken cancellationToken);
+        Task<ImmutableArray<CodeFixCollection>> GetFixesAsync(Document document, string diagnosticId, TextSpan textSpan, CancellationToken cancellationToken);
```

Comment:
this actually return 1 fix all for the diagnostic right? there is all these cases where what if multiple analyzer return same diagnostic id, what if there are mutliple way to fix same diagnostic id and etc that I talked about.

so, we probably need some doc saying this doesn't cover all cases.

Reply:
add https://github.com/dotnet/roslyn/issues/27066